components/UI/FilterMenu/FilterReasonMenu.tsx

```diff
@@ -56,7 +54,7 @@ const ReasonFilterMenu: React.FC<FilterReasonMenuProps> = ({ onChange }) => {
   };
 
   useEffect(() => {
-    onChange(selectedValue);
+    onChange(selectedValue as any);
```

Comment:
why the type cast? also we can enable the following rule again, we are no using any extra props that we need to not listen for.